### PR TITLE
Get local env to a workable state

### PR DESCRIPTION
**Pull request by nathf**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,11 +32,13 @@
     "filesize": "3.3.0",
```
Comment (maxious) [inline package.json]: Need to add
```
 "chokidar": "^1.0.0",
```
for yarn users because of babel-cli.

File: src/widget/CaseStudy.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ const store = createStore(initialState)
 
 ReactDOM.render(
   <Provider store={store}>
-    <BrowserRouter>
+    <BrowserRouter basename={window.__ROUTER_BASENAME__}>
```
Comment (matt-sm) [inline src/widget/CaseStudy.js]: what is this?

Comment (nathf) [inline src/widget/CaseStudy.js]: Temporary workaround for the local dev harness

It deserve a `FIXME`

**Comment by matt-sm** [comment]
Update readme?

**Pull request merged**